ui/packages/app/src/components/Integer/Integer.svelte

```diff
@@ -0,0 +1,33 @@
+<script lang="ts">
```

Comment:
This pattern may not be sustainable if we also want to introduce more numeric types in the future like natural number. Pretty new to svelte so this is the first thing I thought of to get the desired behavior.

Reply:
I don't think we need and Integer component in frontend, we can make use of `Number.integer` using its `get_template_context` and adopt the Number component's input filtering in Frontend according to that.
cc: @abidlabs @aliabid94

Reply:
Instead of creating a new frontend component, we can just utilize `get_template_context` function.

Reply:
Thank you for the pointer. Will look into this!